packages/wrangler/src/queues/cli/commands/create.ts

```diff
@@ -1,25 +1,65 @@
 import { readConfig } from "../../../config";
 import { logger } from "../../../logger";
+import type { CreateQueueBody } from "../../client";
 import { createQueue } from "../../client";
 import type {
 	CommonYargsArgv,
 	StrictYargsOptionsToInterface,
 } from "../../../yargs-types";
+import { UserError } from "../../../errors";
+import { handleFetchError } from "../../utils";
 
 export function options(yargs: CommonYargsArgv) {
-	return yargs.positional("name", {
-		type: "string",
-		demandOption: true,
-		description: "The name of the queue",
-	});
+	return yargs
+			.positional("name", {
+				type: "string",
+				demandOption: true,
+				description: "The name of the queue",
+			})
+		.options({
+			"delivery-delay": {
+				type: "number",
+				describe: "How long a published messages should be delayed for, in seconds. Must be a positive integer",
+			},
+			"no-delivery-delay": {
+				type: "boolean",
+				describe: "Sets published messages to have no delay",
+				boolean: true,
+			}
+		});
+}
+
+function createBody(args: StrictYargsOptionsToInterface<typeof options>): CreateQueueBody {
+	const body: CreateQueueBody = {
+		queue_name: args.name
+	}
+
+	// Workaround, Yargs does not play nicely with both --parameter and --no-parameter set.
```

Comment:
I just did some googling and found: https://yargs.js.org/docs/#api-reference-conflictsx-y

Have you tried using `conflicts`? (if we actually keep both options...)

Reply:
Didn't use the right keywords to find that one. Will take a look.

Reply:
Didn't work. Apparently "no-x" has a special meaning, meaning negation of x. It's not viewed as a separate parameter, it's viewed as x.

Reply:
OK no worries, we don't need to use it